metricbeat/module/windows/service/service_windows.go

```diff
@@ -60,12 +75,34 @@ var serviceStates = map[ServiceState]string{
 	ServiceStopped:         "Stopped",
 }
 
-var serviceStartTypes = map[ServiceStartType]string{
-	ServiceAutoStart:   "Automatic",
-	ServiceBootStart:   "Boot",
-	ServiceDemandStart: "Manual",
-	ServiceDisabled:    "Disabled",
-	ServiceSystemStart: "System",
+type serviceStartType int
+
+const (
+	Boot = iota
+	System
+	Automatic
+	Manual
+	Disabled
+	AutomaticDelayed
+	AutomaticTriggered
+	AutomaticDelayedTriggered
+	ManualTriggered
+)
+
+var serviceStartTypes = [...]string{
+	"Boot",
+	"System",
+	"Automatic",
+	"Manual",
+	"Disabled",
+	"Automatic (Delayed)",
+	"Automatic (Triggered)",
+	"Automatic (Delayed, Triggered)",
+	"Manual (Triggered)",
+}
+
+func (startType ServiceStartType) String() string {
+	return serviceStartTypes[startType]
```

Comment:
An invalid value can causes an panic because there is no bounds checking. Using a map for the strings will fix it.